### DISCUSSION TITLE
Daily standup (Friday, March 17th)

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#59] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by Int-Angel** [comment]
**What did you achieve in the last 24 hours?**:
 - I finished my requirements specification and completed the quality plan

**What are your priorities for the next 24 hours?**:
 - deliver the requirements document and the project plan

**Blockers**:
 - none

**Shoutouts**:
 -

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on quality plan and added new requirement to the document

**What are your priorities for the next 24 hours?**:
 - Finish adding requirements to project

**Blockers**:
 - None

**Shoutouts**:
 - @Int-Angel for being so dependable

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Added requirements updates to matrix

**What are your priorities for the next 24 hours?**:
 - Finish the project _calendar_.
 - Add my updates to GH backlog.
 - Meet with teacher to address our low score on the requirements document.

**Blockers**:
 - None

**Shoutouts**:
 - @Int-Angel @DavBot02 @Jhoscar22 @paotg123 for finishing the quality plan yesterday!

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
-Added requirements updates to matrix

**What are your priorities for the next 24 hours?**:
review and finish assignments

Blockers:
None

Shoutouts:

@Int-Angel @antonieto good leaders

**Comment by Jhoscar22** [comment]
What did you achieve in the last 24 hours?:
-Added requirements updates to matrix, specifications document and project backlog in GitHub.

What are your priorities for the next 24 hours?:
-Make sure everything is finished and delivered in time.
-Provide help to classmates if necessary

Blockers:
-Tiredness

Shoutouts:

@Int-Angel @antonieto @DavBot02 @paotg123 for completing all of the deliverables for this period.
